ocs_ci/helpers/dr_helpers.py

```
@@ -2231,11 +2231,24 @@ def get_managed_cluster_node_ips():
-    ][0]
+    dr_cluster_relations = config.MULTICLUSTER.get("dr_cluster_relations", [])
+    if dr_cluster_relations:
+        # dr_cluster_relations is a list containing cluster pairs list with the index 0 pair as current pair under test
+        dr_cluster_names = dr_cluster_relations[0]
+        primary_name = config.get_cluster_name_by_index(primary_index)
+        secondary_index = config.get_cluster_index_by_name(
+            [
+                cluster_name
+                for cluster_name in dr_cluster_names
+                if cluster_name != primary_name
+            ][0]
```

Ruff (0.15.21)

[warning] Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)